src/main/kotlin/no/nav/k9/los/K9Los.kt

```diff
@@ -276,13 +277,15 @@ fun Application.k9Los() {
             route("mock") {
                 MockGrensesnitt()
             }
-            route("innsikt") {
+            route("innsyn") {
                 innsiktGrensesnitt()
+                innsynApis()
             }
         } else {
             authenticate(*issuers.allIssuers()) {
-                route("innsikt") {
+                route("innsyn") {
```

Comment:
Det er kanskje bedre med `drift` hvis vi skal rename den? Innsyn er allerede et begrep brukt mot innsyn i sak nav.no
Du må endre det til det samme i frontend. Hvis ikke kan det ikke nås lenger

Reply:
ja, lurt. Qadeer foreslo forvaltning :thinking: 
Jeg var på nippet til å kalle det for "kikkehull"